src/main/resources/site/parts/searchResult/searchResultView.jsx

```diff
@@ -241,8 +241,12 @@ function SearchResult(props) {
             <Title>{props.title}</Title>
             <Input
               size="lg"
-              value={searchTerm} handleChange={setSearchTerm} searchField
-              submitCallback={goToSearchResultPage}></Input>
+              value={searchTerm}
+              handleChange={setSearchTerm}
+              searchField
+              submitCallback={goToSearchResultPage}
+              ariaLabelWrapper={props.term ? props.mainSearchPhrase : undefined}
```

Comment:
Er det noen grunn til at man kun legger det på om man har søkt på noe?

Reply:
Når man først går inn på søketreffsiden så er søkefeltet i headeren fremdeles synlig, den har aria-label="Hovedsøk". 
Det er satt for at den ikke skal rendres når brukeren søker med søkefeltet i søketreffsiden, og da får det store søkefeltet aria-label="Hovedsøk" siden den blir det eneste søkefeltet på siden.